packages/Upphandling/screens/DIS.js

```diff
@@ -176,8 +188,14 @@ export const DIS = ({ navigation, route }) => {
         bottomOffset={350}
       >
         <View style={styles.bottomDrawerContent}>
-          <Text category="label" style={styles.footerLabel}>{selected.length} ärenden valda. Gå vidare och skapa upphandling.</Text>
-          <Button onPress={() => navigation.navigate('CreateTender', { id, issues: selected })}>
+          <Text category="label" style={styles.footerLabel}>
```

Comment:
When this box is active there is also black overlay in top of the list of issues from github. If you want to select 8 github issues you would have to click 15 times. 8 to check boxes and 7 in between to click away the black overlay. Is this on purpose to try and game people into creating smaller DISes or a UX mishap?